### PR TITLE
Backend: CRUD mutations for winter resources

### PR DESCRIPTION
**Pull request by haavikko**
### Closes :no_good_woman:
**[VEN-409](https://helsinkisolutionoffice.atlassian.net/browse/VEN-409):**

### CODE REVIEW

**Review by a-rmz — CHANGES_REQUESTED** [review]
Before digging deep into the code (will do meanwhile), can you please rebse the styling commits to the commit where you made those changes? We try to keep a clean commit history, so don't be affraid to use rebase/squash/force-push. :)

Additionally, I don't know if comes out of the box, but we have [`pre-commit`](https://pre-commit.com/) setup with few hooks to avoid having this "style fixing" commits. In case it doesn't, I recommend you take a look.

**Comment by haavikko** [comment]
> Before digging deep into the code (will do meanwhile), can you please rebse the styling commits to the commit where you made those changes? We try to keep a clean commit history, so don't be affraid to use rebase/squash/force-push. :)
> 
> Additionally, I don't know if comes out of the box, but we have [`pre-commit`](https://pre-commit.com/) setup with few hooks to avoid having this "style fixing" commits. In case it doesn't, I recommend you take a look.

I just did rebase/squash/force-push and merged all commits in the PR.
I'll look into the pre-commit hook, thanks for the heads-up

File: resources/schema.py [hunk 1]
```diff
@@ -948,6 +1330,14 @@ def resolve_winter_storage_areas(self, info, **kwargs):
         ).select_related("availability_level", "municipality")
 
 
+IMAGE_FILE_DESCRIPTION = (
+    "The `imageFile` field takes an image as input. "
+    "To provide the file, you have to perform the request with a client "
+    "that conforms to the [GraphQL Multipart Request Spec]"
+    "(https://github.com/jaydenseric/graphql-multipart-request-spec)."
+)
```
Comment (a-rmz) [inline resources/schema.py:1338]: Let's also get rid of this since we don't use the image upload anymore.

**Review by a-rmz — COMMENTED** [review]
To make the code more readable, could you please split it as the [`payments/schema`](https://github.com/City-of-Helsinki/berth-reservations/tree/master/payments/schema)?

File: resources/schema.py (hunk 1 above)
Comment (haavikko) [reply resources/schema.py:1338]: fixed

**Comment by a-rmz** [comment]
@haavikko not sure if this got lost with all the comments, but:

> To make the code more readable, could you please split it as the [`payments/schema`](https://github.com/City-of-Helsinki/berth-reservations/tree/master/payments/schema)?

File: resources/schema.py [hunk 2]
```diff
@@ -709,13 +730,171 @@ def mutate_and_get_payload(cls, root, info, **input):
...
+        delete_inactive_leases(BerthLease, Q(berth__pier__harbor=harbor), "Harbor")
 
         harbor.delete()
 
         return DeleteHarborMutation()
 
 
+class WinterStorageAreaInput(AbstractAreaInput):
+    municipality_id = graphene.String()
+    image_file = Upload()
```
[the remark this answers is not in the document]
Comment (haavikko) [reply resources/schema.py]: > To make the code more readable, could you please split it as the [`payments/schema`](https://github.com/City-of-Helsinki/berth-reservations/tree/master/payments/schema)?

done

[the remark this answers is not in the document]
Comment (haavikko) [reply resources/schema.py]: > This is not used anymore, so it can be removed (also from `HarborInput`)

done

**Comment by haavikko** [comment]
> @haavikko not sure if this got lost with all the comments, but:
> 
> > To make the code more readable, could you please split it as the [`payments/schema`](https://github.com/City-of-Helsinki/berth-reservations/tree/master/payments/schema)?

I was working on it, just pushed changes

**Review by a-rmz — APPROVED** [review]
Really nice job!! 🏆 🥇

**Pull request merged**